### PR TITLE
Profile pet

### PR DESCRIPTION
**Pull request by VincentMomot**
Still finishing this page, but wanted the most up-to-date page. If you know how to add the activities, you can give that a try, or I'll do that tomorrow!

### CODE REVIEW

File: routes/homepageRoutes.js [hunk 1]
```diff
@@ -42,12 +42,5 @@ router.get('/signuppage', async (req, res) => {
   res.render('signup');
 });
 
-router.get('api/pet', async(req,res) => {
```
Comment (jstndhouk) [inline routes/homepageRoutes.js]: I believe we need this for the homepage or view pets page to load

**Review by jstndhouk — DISMISSED** [review]
Take a look at my comment for routes/homepageRoutes.js

**Review by VincentMomot — COMMENTED** [review]
I added the code back for the allpets data

File: routes/homepageRoutes.js (hunk 1 above)
Comment (sistaniabong) [reply routes/homepageRoutes.js]: we don't need this route in the homepage, because this is under /api route.

**Review by Tmcrocker89 — APPROVED** [review]

**Review by sistaniabong — APPROVED** [review]

**Pull request merged**